### PR TITLE
Fixed #35152 -- Added en_CA locale formats.

### PR DESCRIPTION
**Pull request by candideu**
Adds Canadian English (en_CA) to locale conf, similar to https://github.com/django/django/commit/114b70ccf8dcd7af86d5cd464d66aa7761400884 and ​https://github.com/django/django/commit/1ac397674b2f64d48e66502a20b9d9ca6bfb579a

Ticket: https://code.djangoproject.com/ticket/35152#ticket

### CODE REVIEW

File: django/conf/global_settings.py [hunk 1]
```diff
@@ -70,6 +70,7 @@ def gettext_noop(s):
     ("en", gettext_noop("English")),
     ("en-au", gettext_noop("Australian English")),
     ("en-gb", gettext_noop("British English")),
+    ("en-ca", gettext_noop("Canadian English")),
```
Comment (felixxm) [inline django/conf/global_settings.py:73]: Swap with `en-gb` to keep it alphabetized.

File: django/conf/locale/en_CA/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (felixxm) [inline django/conf/locale/en_CA/__init__.py:1]: Make this file empty.

File: django/conf/locale/fr_CA/formats.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 MONTH_DAY_FORMAT = "j F"
 SHORT_DATE_FORMAT = "Y-m-d"
 SHORT_DATETIME_FORMAT = "Y-m-d H\xa0h\xa0i"
-FIRST_DAY_OF_WEEK = 1  # Lundi
+FIRST_DAY_OF_WEEK = 0  # dimanche
```
Comment (felixxm) [inline django/conf/locale/fr_CA/formats.py:12]: @charettes Can you check this change?

File: django/conf/locale/__init__.py [hunk 4]
```diff
@@ -134,6 +134,12 @@
         "name": "British English",
         "name_local": "British English",
     },
+    "en-ca": {
```
Comment (felixxm) [inline django/conf/locale/__init__.py:137]: Swap with `en-gb`.

File: django/conf/locale/fr_CA/formats.py (hunk 3 above)
Comment (charettes) [reply django/conf/locale/fr_CA/formats.py:12]: It seems to be ambiguous, I've seen many instances in Québec which has the majority of `fr_CA` users refer to Monday / _Lundi_ as the first day of the week but from looking at other locales and documentation online it seems that the agreement is around Sunday / _Dimanche_ instead.

👍 on chaning

Comment (felixxm) [reply django/conf/locale/fr_CA/formats.py:12]: Thanks :+1:

**Comment by felixxm** [comment]
@candideu Do you have a colleague who could confirm format definitions for `en_CA`?

**Comment by candideu** [comment]
> @candideu Do you have a colleague who could confirm format definitions for `en_CA`?

Hi @felixxm I'll ask some Canadian colleagues to review later today.

Also, this info was hidden in one of the commits, but I sourced the formatting rules from:
- https://www.noslangues-ourlanguages.gc.ca/en/writing-tips-plus/numbers-decimal-fractions.html
- https://www.noslangues-ourlanguages.gc.ca/en/writing-tips-plus/time-of-day-elapsed-time.html
- https://github.com/django/django/commit/1ac397674b2f64d48e66502a20b9d9ca6bfb579a

**Review by buhama — APPROVED** [review]
Config for en_CA LGTM

File: django/conf/global_settings.py (hunk 1 above)
Comment (felixxm) [reply django/conf/global_settings.py:72]: OK, we don't have a separate translations for `en_CA` so it should be enough to add `formats.py`.

**Review by felixxm — COMMENTED** [review]
@candideu Thanks :+1: Welcome aboard :boat:

**Comment by candideu** [comment]
Thank you @felixxm !

**Pull request merged**